openapiart/bundler.py

```diff
@@ -1060,6 +1060,68 @@ def _generate_value_schema(
                 metric_tags_schema_name
             ] = metric_tags_schema
 
+        if "features" in xpattern and "random" in xpattern["features"]:
+            if xpattern["format"] in ["integer", "ipv4", "ipv6", "mac"]:
+                random_pattern_name = "{}.Random".format(schema_name)
+                schema["properties"]["choice"]["x-enum"]["random"] = {
+                    "x-field-uid": 6
+                }
+                schema["properties"]["random"] = {
+                    "$ref": "#/components/schemas/{}".format(
+                        random_pattern_name
+                    )
+                }
+                schema["properties"]["random"]["x-field-uid"] = auto_field.uid
+                random_auto_field = AutoFieldUid()
+                random_schema = {
+                    "description": "{} random pattern".format(
+                        xpattern["format"]
+                    ),
+                    "type": "object",
+                    "properties": {
+                        "min": {
+                            "description": "The minimum possible value generated by the random value generator.",
+                            "type": type_name,
+                            "x-field-uid": random_auto_field.uid,
+                        },
+                        "max": {
+                            "description": "The maximum possible value generated by the random value generator.",
+                            "type": type_name,
+                            "x-field-uid": random_auto_field.uid,
+                        },
+                        "seed": {
+                            "description": "The seed value to initialize the random value generator to a deterministic state.",
```

Comment:
I would suggest that we default the seed value to 0, but treat this as a special case which causes the seed to be set to a random value. This would mimic the behavior of random functions in software libraries. But it's not crucial since the script can always generate a random number to use

Reply:
We will update model description to treat 0 as special seed value (which will be replaced by a random value in the implementation to get true randomness). But we want to set default seed value to 1 so that test cases by default generates predictable sequence, which is easy to debug, for any failure. The user can opt for a seed value of 0 if they want different behavior.